app/helpers/mustache_helper.rb

```diff
@@ -251,9 +223,7 @@ def navigation_global
             :text  => "My Europeana"
           }
         ]
-    },
-
-    :footer  => common_footer
+    }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.